classes/services/IssueService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file classes/services/IssueService.php
+*
+* Copyright (c) 2014-2017 Simon Fraser University
+* Copyright (c) 2000-2017 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* @class IssueService
+* @ingroup services
+*
+* @brief Helper class that encapsulates issue business logic
+*/
+
+namespace App\Services;
+
+class IssueService {
+
+	/**
+	 * Determine if a user can access galleys for a specific issue 
+	 * 
+	 * @param \Journal $journal
```

Comment:
Use application-impartial lingo in pkp-lib (`Context` / `$context` instead of `Journal`/`$journal`).